zebra-chain/src/serialization/compact_size.rs

```diff
@@ -0,0 +1,350 @@
+//! Types and serialization for the Bitcoin `CompactSize` format.
+//!
+//! Zebra decodes `CompactSize` fields into two different types,
```

Comment:
Nit: Feel free to ignore this, but I was wondering if we should standardize on using "`CompactSize`" or "compactSize" inside the documentation :thinking:

Reply:
The ZIPs use `CompactSize`, so I'll follow them:
https://zips.z.cash/zip-0155#specification

Fixed in 02ec95bfc.